src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -396,55 +396,55 @@ public async Task GetSemanticTokens_HalfOfCommentAsync()
         {
             var txt = $"@addTagHelper *, TestAssembly{Environment.NewLine}@* comment";
 
-            await AssertSemanticTokens(txt, isRazor: true);
+            await AssertSemanticTokensAsync(txt, isRazor: true);
         }
 
         [Fact]
         public async Task GetSemanticTokens_NoAttributesAsync()
         {
             var txt = $"@addTagHelper *, TestAssembly{Environment.NewLine}<test1></test1> ";
 
-            await AssertSemanticTokens(txt, isRazor: false);
+            await AssertSemanticTokensAsync(txt, isRazor: false);
         }
 
         [Fact]
         public async Task GetSemanticTokens_WithAttributeAsync()
         {
             var txt = $"@addTagHelper *, TestAssembly{Environment.NewLine}<test1 bool-val='true'></test1> ";
 
-            await AssertSemanticTokens(txt, isRazor: false);
+            await AssertSemanticTokensAsync(txt, isRazor: false);
         }
 
         [Fact]
         public async Task GetSemanticTokens_MinimizedAttributeAsync()
```

Comment:
I don't feel strongly but I do lean more preferably towards test projects not requiring "async" suffixes on test names.

Reply:
➡️ Test projects do not require the `Async` suffix on test names. Roslyn.Diagnostics.Analyzers contains a diagnostic suppressor which suppresses VSTHRD200 on any test method.